Crashlytics/Crashlytics/Settings/FIRCLSSettingsOnboardingManager.m

```diff
@@ -22,20 +22,17 @@
 #import "Crashlytics/Crashlytics/Models/FIRCLSSettings.h"
 #import "Crashlytics/Crashlytics/Settings/Models/FIRCLSApplicationIdentifierModel.h"
 #import "Crashlytics/Crashlytics/Settings/Operations/FIRCLSDownloadAndSaveSettingsOperation.h"
-#import "Crashlytics/Crashlytics/Settings/Operations/FIRCLSOnboardingOperation.h"
 #import "Crashlytics/Shared/FIRCLSConstants.h"
 #import "Crashlytics/Shared/FIRCLSNetworking/FIRCLSFABNetworkClient.h"
 #import "Crashlytics/Shared/FIRCLSNetworking/FIRCLSURLBuilder.h"
 
-@interface FIRCLSSettingsOnboardingManager () <FIRCLSDownloadAndSaveSettingsOperationDelegate,
-                                               FIRCLSOnboardingOperationDelegate>
+@interface FIRCLSSettingsOnboardingManager () <FIRCLSDownloadAndSaveSettingsOperationDelegate>
```

Comment:
So this one's kindof a doozy, but can we:
 - Rename this class to just the `FIRCLSSettingsManager`?
 - Rename variable references to `settingsManager`, eg: https://github.com/firebase/firebase-ios-sdk/blob/master/Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m#L175
 - Rename this method: https://github.com/firebase/firebase-ios-sdk/blob/master/Crashlytics/Crashlytics/Controllers/FIRCLSReportManager.m#L457

Highly recommend the Xcode tool: "Right Click > Refactor > Rename" for this

Reply:
Main reason is onboarding is a confusing concept because of it's ridiculousness, and after this change we can draw a better parallel between SettingsManager and ReportManager concepts